### ISSUE TITLE
Duping itens with Connectors and Grinders

### ISSUE THREAD

**Issue by Mangrulio** [DraconisCluster/DX-Issues#51]
** tl; dr **
--type of issue (bug, fr, etc), short what and why--
A player opened a ticket telling that when you eject an item from a connector and you grind it, it will dupe once for every grinders working on it. Using sorters it will do it at industrial levels.
** Full info **
--long what and why--

**Screenshots**
--If applicable, add screenshots to help explain your problem.--
![image](https://github.com/DraconisCluster/DX-Issues/assets/167803963/93363e8b-8011-4fe9-a1f2-50198578cfce)
![image](https://github.com/DraconisCluster/DX-Issues/assets/167803963/8edbef24-701b-4f60-8990-878a5899056b)
![image](https://github.com/DraconisCluster/DX-Issues/assets/167803963/5e67bf66-392e-471b-be78-10e9945945b8)

**Comment by draconb** [comment]
Resolved by removing TC free floating pickup
